pkgs/build-support/fetchdocker/default.nix

```diff
@@ -22,8 +22,8 @@ assert null == lib.findFirst (c: "/"==c) null (lib.stringToCharacters repository
 assert null == lib.findFirst (c: "/"==c) null (lib.stringToCharacters imageName);
 
 let
-  # Abuse `builtins.toPath` to collapse possible double slashes
-  repoTag0 = builtins.toString (builtins.toPath "/${stripScheme registry}/${repository}/${imageName}");
+  # Abuse paths to collapse possible double slashes
+  repoTag0 = builtins.toString (/. + "/${stripScheme registry}/${repository}/${imageName}");
```

Comment:
I think this can just be done with replaceStrings:

```
repoTag0 = replaceStrings ["//"] ["/"] "/${stripScheme registry}/${repository}/${imageName}";
```

Reply:
No problem.